viv-wallet-app/src/router/index.ts

```diff
@@ -16,6 +16,11 @@ const routes = [
         name: "wallet",
         component: () => import(/* webpackChunkName: "wallet" */ "../views/Wallet.vue")
     },
+    {
+        path: "/history",
```

Comment:
this path is weird... which history is that?

Reply:
on the initial issue, I said to put it on `/home` because basically every user has a home, but we display different things for different people.

Reply:
Why not. I have a lack of inspiration.

Reply:
We could say "Mes actions" because it's an action history (containing all the records).

Reply:
But it's not "my" actions...

Reply:
True

Reply:
"Historique des actions"